src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<blockquote>The ratio $(MTOW/S_w)_{data}$ is called the wing loading</blockquote><p>"and is an input of the module".</p><p> </p><p>Also, I know I am mainly responsible for that, but we need to be more consistent about the usage of words "model" and "module". Maybe we should even stop using "module". In all cases, I think the good word for all this part should be only "model" (but we can discuss that)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Changed in 3c61c6ae
Commit description explains the reasoning behind choice of module/model

Reply:
Corrected in 3c61c6ae

Reply:
<p>I agree with the rationale.</p>